Formula/odo.rb

```diff
@@ -1,7 +1,7 @@
 class Odo < Formula
   desc "OpenShift Command-line for Developers"
   homepage "https://github.com/redhat-developer/odo"
-  url "https://github.com/redhat-developer/odo/archive/v0.0.10.tar.gz"
+  url "https://github.com/redhat-developer/odo/archive/v0.0.11.tar.gz"
   sha256 "cb810e6e125b7bd5cd9d5ca6ef94f2b188260063f98a59e88d8bc7101cf1ebac"
```

Comment:
What is this hex supposed to be?
The commit id of latest commit on Odo?
Cc: @kadel @surajnarwade @cdrage